src/common/components/card/Card.tsx

```diff
@@ -57,10 +57,14 @@ const Card: FunctionComponent<CardProps> = ({
       </div>
 
       <div className={styles.end}>
-        <button aria-label={actionText} className={styles.actionWrapper}>
+        <Button
+          variant="supplementary"
+          aria-label={actionText}
```

Comment:
The same text is used inside this component and in the aria label. Won't this result in the same string being read twice?

Reply:
Good point! Leftover from old sins.